RecoLocalCalo/HGCalRecAlgos/interface/HGCalImagingAlgo.h

```diff
@@ -195,6 +213,9 @@ hgcal::RecHitTools rhtools_;
 // The algo id
 reco::CaloCluster::AlgoId algoId_;
 
+// For keeping the density per hit
+ Density density_v;
```

Comment:
Why ```density_v``` and not ```density_```, as for the other local members?
(Later on there is also a ```v_sigmaNoise_``` which could/should be renamed ```sigmaNoise_```, isn't it?)